### DISCUSSION TITLE
2023/08/04/230804/

### DISCUSSION THREAD

**Discussion by giscus** [cloudmatelabs/tech.cloudmt.co.kr#170] [Blog Comments]
# 2023/08/04/230804/

자바스크립트로 프로젝트를 진행하면서 여러가지 api와 통신을 하는 경우가 많았고 api통신을 할 때 응답된 데이터를 다루다보면서 자연스럽게 자바스크립트의 비

https://tech.cloudmt.co.kr/2023/08/04/230804/

**Comment by 9z** [comment]
function first(callback) {
  setTimeout(function () {
    console.log("첫번째");
    callback();
  }, 3000);
}

function second() {
  console.log("두번째");
}

first(function () {
  second();
});

이 코드에서 마지막에 함수 호출할떄

first(second); 이렇게 해도 되지 않나요?
함수로 한번더 감싸준 이유가 궁금합니다!
